user_groups/user_groups.md

```diff
@@ -2,11 +2,11 @@
 
 [Get a user group list](#get-a-user-group-list)  
 [Get a user group](#get-a-user-group)  
-[Get users for a user group](#get-users-for-a-user-group) <br />
-[Add a user to a user group](#add-a-user-to-a-user-group) <br />
-[Remove a user from a user group](#remove-a-user-from-a-user-group) <br />
-[Get user group permissions](#get-user-group-permissions) <br />
-[Add or update user group permission for a given resource](#add-or-update-user-group-permission-for-a-given-resource) <br />
+[Get users for a user group](#get-users-for-a-user-group)\
+[Add a user to a user group](#add-a-user-to-a-user-group)\
+[Remove a user from a user group](#remove-a-user-from-a-user-group)\
```

Comment:
Why do you need this symbol "\" @WMKhant ?

Reply:
I add those to move to the new line. It should have been done automatically. Git become buggy so I put them on the line that new line did not happen.

Reply:
removed and fixed the titles. Just need to use double space at the end. haha